network/p2p/acp118/handler_test.go

```diff
@@ -143,18 +142,31 @@ func TestHandler(t *testing.T) {
 
 // The zero value of testVerifier allows signing
 type testVerifier struct {
-	Errs []*common.AppError
+	t            *testing.T
+	errs         []*common.AppError
+	clientNodeID ids.NodeID
+}
+
+func newTestVerifier(t *testing.T, clientNodeID ids.NodeID, errs []*common.AppError) *testVerifier {
+	return &testVerifier{
+		t:            t,
+		errs:         errs,
+		clientNodeID: clientNodeID,
+	}
 }
 
 func (t *testVerifier) Verify(
-	context.Context,
-	*warp.UnsignedMessage,
-	[]byte,
+	_ context.Context,
+	_ *warp.UnsignedMessage,
+	justification []byte,
+	nodeID ids.NodeID,
 ) *common.AppError {
-	if len(t.Errs) == 0 {
+	require.Equal(t.t, t.clientNodeID, nodeID)
+	require.Equal(t.t, []byte("justification"), justification)
```

Comment:
I think we should pass in the expected justification instead of hard-coding here